libr/util/rxml.c

```diff
@@ -249,7 +249,7 @@ R_API void r_xml_free(RXml *x) {
 R_API RXmlRet r_xml_parse(RXml *x, int _ch) {
 	/* Ensure that characters are in the range of 0..255 rather than -126..125.
 	 * All character comparisons are done with positive integers. */
-	ut8 ch = (ut8 )(_ch + 256) & 0xff;
+	unsigned ch = (unsigned)(_ch + 256) & 0xff;
```

Comment:
```suggestion
	ut8 ch = (ut8)((_ch + 256) & 0xff);
```

Reply:
i think it's better to specify the type, dspite "unsigned" is an alias for "unsigned int" this way it's clearer imho can you confirm this patch works?

Reply:
Agreed. It was `ut32` before, but then I noticed that `r_xml_refend()` uses `unsigned`. Will change it back.

Reply:
Changed to `ut32`, and confirmed working:

<img width="868" alt="Screenshot 2022-12-15 at 18 21 21" src="https://user-images.githubusercontent.com/735197/207926494-87d5349f-7db5-4bae-a0f7-dfb927992feb.png">